test/test_utils.py

```diff
@@ -72,3 +75,23 @@ def print_test_update(device, accuracy):
     accuracy: Float.
   """
   print('[{}] Accuracy={:.2f}%'.format(device, accuracy))
+
+
+def should_report_lr(current_device, devices, machine_ordinal_num):
```

Comment:
This should be called differently. Like `is_first_device()` so that it can be used in other places where we want to do one-device actions.

Reply:
Switched and updated the docstring of is_first_device